pkgs/by-name/am/amber-lang/package.nix

```
@@ -0,0 +1,66 @@
+{ lib,
+  pkgs,
+  fetchFromGitHub,
+  rustPlatform,
+  bc,
+  curl,
+  bash,
+  runCommand,
+  amber-lang
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "amber-lang";
+  version = "0.3.1-alpha";
+
+  src = fetchFromGitHub {
+    owner = "Ph0enixKM";
+    repo = "Amber";
+    rev = "${version}";
```

Its already a string, I don't believe you need to use `${}`
```suggestion
    rev = version;
```

---

Nice catch!